src/ws/cockpit.service.in

```diff
@@ -9,7 +9,7 @@ After=cockpit-wsinstance-http.socket cockpit-wsinstance-http-redirect.socket coc
 RuntimeDirectory=cockpit/tls
 # systemd ≥ 241 sets this automatically
 Environment=RUNTIME_DIRECTORY=/run/cockpit/tls
-ExecStartPre=+@sbindir@/remotectl certificate --ensure --user=root --group=@group@ --selinux-type=@selinux_config_type@
+ExecStartPre=+@libexecdir@/cockpit-certificate-ensure
```

Comment:
Yay! Killing remotectl will then be a follow-up PR, I suppose?

Reply:
Yup.